data/strings/strings.txt

```diff
@@ -20153,93 +20153,95 @@
+    sw = Tumia utafutaji au uguse kwenye ramani ili kuongeza mahali pa kuanzia
+    th = ใช้การค้นหาหรือแตะบนแผนที่เพื่อเพิ่มจุดเริ่มต้นเส้นทาง
+    tr = Bir rota başlangıç noktası eklemek için aramayı kullanın veya haritaya dokunun
+    uk = Скористайтеся пошуком або торкніться мапи, щоб додати початкову точку маршруту
+    vi = Sử dụng tìm kiếm hoặc nhấn vào bản đồ để thêm điểm bắt đầu tuyến đường
+    zh-Hans = 使用搜索或点击地图添加路线起点
+    zh-Hant = 使用搜尋或點擊地圖新增路線起點
 
   [routing_add_finish_point]
     tags = android,ios
-    en = Add a destination to plan a route
-    af = Voeg ’n rigting by om ’n roete te beplan
-    ar = أضف النهاية لتخطيط المسار
-    az = Marşrutu planlaşdırmaq üçün təyinat əlavə edin
-    be = Дадайце пункт прызначэння, каб пракласці маршрут
-    bg = Добавяне на крайна точка за планиране на маршрут
-    ca = Afegeiu la destinació per a planificar la ruta
-    cs = Zadejte cílový bod pro plánování trasy
-    da = Tilføj slutpunkt for at planlægge en rute
-    de = Fügen Sie ein Ziel hinzu, um eine Route zu planen
-    el = Προσθέστε τελικό προορισμό, για να σχεδιάσετε μια διαδρομή
-    es = Buscar un destino para el plan de ruta
-    et = Marsruudi planeerimiseks lisa sihtkoht
-    eu = Gehitu helmuga bat ibilbide bat antolatzeko
-    fa = یک مقصد برای برنامه ریزی یک مسیر اضافه کنید
-    fi = Lisää määränpää reitin suunnittelua varten
-    fr = Ajouter un point d'arrivée pour planifier un itinéraire
-    he = יש להוסיף יעד כדי לתכנן מסלול
-    hi = मार्ग की योजना बनाने के लिए एक गंतव्य जोड़ें
-    hu = Adjon hozzá végpontot az útvonal tervezéséhez
-    id = Tambahkan titik akhir untuk merencanakan rute
-    it = Aggiungi un punto di arrivo per pianificare un percorso
-    ja = ルートを計画するには目的地を追加してください
-    ko = 경로 계획을 세우기 위한 끝 지점을 추가하세요
-    lt = Pridėkite tikslo tašką, kad planuotumėte maršrutą
-    mr = मार्ग नियोजित करण्यासाठी इष्टस्थान जोडा
-    nb = Angi sluttpunkt for å planlegge rute
-    nl = Voeg eindpunt toe om een route te plannen
-    pl = Aby zaplanować trasę, dodaj punkt końcowy
-    pt = Adicionar final da viagem para planear uma rota
-    pt-BR = Adicionar ponto final da viagem para planejar uma rota
-    ro = Adaugă un punct de sosire pentru a planifica un traseu
-    ru = Добавьте конечную точку, чтобы построить маршрут
-    sk = Pridaním cieľového bodu naplánujete trasu
-    sv = Lägg till slutpunkt för att planera en rutt
-    sw = Ongeza eneo la kumalizia ili kupanga njia ya kufuata
-    th = เพิ่มจุดสิ้นสุดเพื่อวางแผนเส้นทาง
-    tr = Rotayı planlamak için bir varış noktası ekleyin
-    uk = Вкажіть кінцеву точку для побудови маршруту
-    vi = Bổ sung điểm kết thúc để lập kế hoạch lộ trình
-    zh-Hans = 添加终点以规划路线
-    zh-Hant = 新增終點以計劃路線
+    comment = User selected the start of a route by pressing Route From. Now the destination of a route should be selected using search or by tapping on the map and then pressing "Route To".
```

Comment:
@organicmaps/translations can you please help with proof-reading this translation in your language (or provide a clearer version)?